samples/csharp/ReferenceApp/README.md

```diff
@@ -97,6 +103,60 @@ To use the DynamoDB Metastore included with the App Encryption library, the foll
  --provisioned-throughput \
    ReadCapacityUnits=1,WriteCapacityUnits=1
 ```
-TODO: Add link to Sceptre template example
 
-TODO: Add multi-region info if/when we handle it  
+#### Global Tables
+
+To use Global Tables, the above table needs to be created with few modifications.
+
+More details about how to create a Global Table can be found in the
+[AWS Developer Guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/globaltables.tutorial.html)
+
+Example run using DynamoDB metastore with key suffixes enabled (for Global Tables), a custom table name, and a
+local DynamoDB endpoint.
+
+```console
+[user@machine ReferenceApp]$ dotnet bin/Debug/netcoreapp2.0/ReferenceApp.dll \
+  --metastore-type DYNAMODB \
+  --key-suffix us-west-2 \
+  --dynamodb-endpoint http://localhost:8000 \
+  --dynamodb-signing-region us-west-2 \
+  --dynamodb-table-name MyGlobalTable
+``` 
+
+## Configuring the Reference App
+Configuration options are provided via command-line arguments. Supported options are as
+follows:
+
+```console
+  -m, --metastore-type           (Default: MEMORY) Type of metastore to use. Enum values: MEMORY, JDBC, DYNAMODB
+
+  -e, --dynamodb-endpoint        The DynamoDb service endpoint (only supported by DYNAMODB)
+
+  -r, --dynamodb-region          The AWS region for DynamoDB requests (only supported by DYNAMODB)
+
+  -s, --dynamodb-signing-region  The DynamoDb service endpoint (only supported by DYNAMODB)
+
+  -t, --dynamodb-table-name      The table name for DynamoDb (only supported by DYNAMODB)
+
+  -x, --key-suffix               Configure the metastore to use key suffixes (only supported by DYNAMODB)
```

Comment:
can we use `-s` instead?

Reply:
this was done when I added `s` for the `singing-region`, and hence `x` seemed to be closest to `suffix`...
anyway, we can keep `s` again now. made the change.